telco-examples/mgmt-cluster/eib/custom/scripts/install-dependencies.sh

```diff
@@ -64,8 +65,11 @@ helm install   metal3 suse-edge/metal3   --namespace metal3-system   --create-na
 
 
 ## install capi
-clusterctl init --core cluster-api:v1.6.0 --infrastructure metal3:v1.6.0
-clusterctl init --bootstrap rke2 --control-plane rke2 --config /home/metal3/clusterctl.yaml
+${KUBECTL} apply -f disable-embedded-capi.yaml
```

Comment:
This is only required if Rancher is deployed.

This is what I'm doing:
```
# If rancher is deployed
if [ $(${KUBECTL} get pods -n ${RANCHERNAMESPACE} -l app=rancher -o name | wc -l) -ge 1 ]; then
  cat <<-EOF | ${KUBECTL} apply -f -
  apiVersion: management.cattle.io/v3
  kind: Feature
  metadata:
    name: embedded-cluster-api
  spec:
    value: false
  EOF

  # Disable Rancher webhooks for CAPI
  ${KUBECTL} delete mutatingwebhookconfiguration.admissionregistration.k8s.io mutating-webhook-configuration
  ${KUBECTL} delete validatingwebhookconfigurations.admissionregistration.k8s.io validating-webhook-configuration
  ${KUBECTL} wait --for=delete namespace/cattle-provisioning-capi-system --timeout=300s
fi
```

Reply:
changed! Thanks